crates/prover/src/lib.rs

```diff
@@ -275,7 +275,8 @@ impl<C: SP1ProverComponents> SP1Prover<C> {
         context.subproof_verifier.replace(Arc::new(self));
         let program = self.get_program(elf).unwrap();
         let opts = SP1CoreOpts::default();
-        let mut runtime = Executor::with_context(program, opts, context);
+        let mut runtime = Executor::with_context_and_elf(program, opts, context, elf);
```

Comment:
Is it feasible to change this to only take in elf? It's a bit weird to take in program + elf.